drivers/network/tdihelpers/tdiconn.c

```diff
@@ -1,15 +1,27 @@
 /*
  * COPYRIGHT:        See COPYING in the top level directory
  * PROJECT:          ReactOS kernel
- * FILE:             drivers/net/afd/afd/tdiconn.c
+ * FILE:             drivers/net/tdihelpers/tdiconn.c
  * PURPOSE:          Ancillary functions driver
  * PROGRAMMER:       Art Yerkes (ayerkes@speakeasy.net)
  * UPDATE HISTORY:
  * 20040708 Created
+ * 20251021 Moved to tdihelpers since it is used by 2 drivers (AFD, NETIO) now
  */
 
 #include <afd.h>
 
+#ifdef UNIMPLEMENTED
+#undef UNIMPLEMENTED
+#endif
+
+/* If you want to see the DPRINT() output in your debugger,
+ * remove the following line (or comment it out):
+ */
+#define NDEBUG 1
```

Comment:
```suggestion
 * remove the following line (or comment it out) */
#define NDEBUG
```
The NDEBUG doesn't require a value.